toolkit/tools/imagecustomizerapi/config.go

```diff
@@ -154,3 +154,74 @@ func (c *Config) IsValid() (err error) {
 func (c *Config) CustomizePartitions() bool {
 	return c.Storage.CustomizePartitions()
 }
+
+func (c *Config) validateUkiPassthroughMode() error {
+	var incompatibleConfigs []string
+
+	// Check for kernel command line modifications
+	if c.OS != nil && len(c.OS.KernelCommandLine.ExtraCommandLine) > 0 {
+		incompatibleConfigs = append(incompatibleConfigs,
+			"os.kernelCommandLine.extraCommandLine: modifies kernel command line embedded in UKI")
+	}
+
+	// Check for verity configuration (modifies initramfs and kernel cmdline)
+	if len(c.Storage.Verity) > 0 {
+		incompatibleConfigs = append(incompatibleConfigs,
+			"storage.verity: adds verity devices which modifies initramfs and kernel cmdline")
+	}
+
+	// Check for verity reinitialization (modifies initramfs and kernel cmdline)
+	if c.Storage.ReinitializeVerity == ReinitializeVerityTypeAll {
+		incompatibleConfigs = append(incompatibleConfigs,
+			"storage.reinitializeVerity: reinitializes verity which modifies initramfs and kernel cmdline")
+	}
+
+	// Check for kernel package changes (would require kernel/initramfs updates)
+	if c.OS != nil && c.OS.Packages.Install != nil {
+		for _, pkg := range c.OS.Packages.Install {
+			if len(pkg) >= 6 && pkg[:6] == "kernel" {
```

Comment:
You can use the [strings.HasPrefix](https://pkg.go.dev/strings#HasPrefix) function.

Reply:
Fixed.